styles.css

```diff
@@ -0,0 +1,502 @@
+    display: flex;
+    flex-direction: column;
+    gap: 2rem;
+    flex: 1; /* Tekst vult resterende ruimte */
+    padding: 3em;
+}
+
+.project-content h3 {
+    color: var(--text-color);
+    font-size: 1.5rem;
+    margin-bottom: 0.5rem;
+
+}
+
+.project-content p {
+    padding-top: 2em;
+    font-size: 0.9em;
+    font-weight: 500;
+    color: var(--text-color);
+    line-height: 1.5;
+}
+
+.projectview {
+    color: var(--main-color);
+    text-transform: uppercase;
+    font-weight: 700;
+    font-size: 0.8em;
+}
+
+/* 6.3.2 Main Info (Company Services) */
+
+.services-wrapper {
+    display: flex;
+    flex-wrap: wrap; /* Zorgt ervoor dat items naar de volgende rij gaan */
+    justify-content: center; /* Center alle blokken horizontaal */
+    gap: 2rem; /* Ruimte tussen de blokken */
+    margin: 10%;
+}
+
+.serviceblock {
+    flex: 1 1 calc(100% / 3 - 2rem); /* 3 kolommen, ruimte aftrekken voor de gap */
+    display: flex;
+    flex-direction: column; /* Zorg dat tekst van boven begint */
+    justify-content: flex-start; /* Logo's e.d. beginnen op dezelfde plek. */
+    align-items: stretch; /* Hoogtes gelijk trekken */
+
+    text-align: left;
+    border-radius: 1%;
+    padding: 1.5em;
+    font-size: 0.8em;
+    border: 1px solid var(--text2-color);
+    overflow: hidden; /* Voorkomt dat content uit het blok loopt */
+    min-height: 250px; /* Optioneel: minimale hoogte instellen */
+}
+
+.serviceblock special {
+    min-height: 300px; /* Optioneel: minimale hoogte instellen */
+}
+
+.serviceblock .img-for-card {
+    display: flex;
+    align-items: start;
+    flex-direction: column;
+    justify-content: flex-start;
+    gap: 0.2em;
+    padding-bottom: 1rem;
+
+}
+
+.serviceblock img {
+    width: 40px;
+    height: 40px;
+}
+
+.serviceblock h3 {
+    margin: 0;
+    font-size: 1.2rem;
+    color: var(--text2-color);
+}
+
+.serviceblock p {
+    font-size: 0.8rem;
+    color: var(--text-color);
+    line-height: 1.2;
+}
+
+/* --------------- 6.4 Contact Form ------------------- */
+
+
+/*    margin-top: 2rem;*/
+/*}*/
+
+/*#contact-form {*/
+/*    display: flex;*/
+/*}*/
+
+.formarea {
+    display: flex;
+    flex-direction: column;
+
+    flex: 1 0 auto;
+    width: 100%;
+}
+
+.form-title-wrapper {
+    display: flex;
+    flex-flow: column;
+    align-items: center;
+}
+
+/*.contactform {*/
+/*    display: flex;*/
+/*    width: 100%;*/
+
+/*    justify-content: center;*/
+/*}*/
+
+.contactform {
+    display: flex;
+    list-style: none;
+    flex-flow: column;
+    align-items: center;
+    margin-top: 2rem;
+}
+
+.form-outer-wrapper {
+
+    flex-flow: column wrap;
+    display: flex;
+    justify-content: center;
+    flex: 1 0 auto;
+    gap: 1rem;
+}
+
+.form-outer-wrapper li {
+    flex: 1 0 50px;
+    max-width: 450px;
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: center;
+
+}
+
+.form-outer-wrapper > li > label {
+    flex: 1 0 450px;
+
+}
+
+.form-outer-wrapper > li > button {
```

Comment:
Als je de container de kaders laat zijn kun je alle labels, inputs en buttons gewoon een `width` of `flex-basis: 100%` geven. Als je dan later een andere breedte wil, hoef je dat maar op één plek te veranderen